devstack_romana.md

```diff
@@ -0,0 +1,93 @@
+# Using Romana on Devstack
+
+When installation completes, some SSH commands are provided for logging into your new hosts, eg:
+```bash
+ssh -i /.../romana-install/romana_id_rsa ubuntu@52.xx.yy.zz
+ssh -i /.../romana/romana-install/romana_id_rsa ubuntu@54.zz.yy.xx
+```
+
+These can be used to log into the host using the generated SSH key. Use the SSH command for the controller to log in.
+
+## Look up information
+
+A `romana` tool has been provided that lets you see details about the setup, and make changes.
+```sh-session
+ubuntu@ip-192-168-99-10:~$ romana show-hosts
+Listing 2 host(s)
+ip-192-168-99-10
+ip-192-168-99-11
+ubuntu@ip-192-168-99-10:~$ romana show-host ip-192-168-99-10
+ip-192-168-99-10       192.168.99.10        10.0.0.1/16     0
+ubuntu@ip-192-168-99-10:~$ romana show-host ip-192-168-99-11
+ip-192-168-99-11       192.168.99.11        10.1.0.1/16     0
+```
+By default, we have two hosts (`192.168.99.10` and `192.168.99.11`). Each host has its own CIDR (`10.0.0.1/16`, `10.1.0.1/16`) that it will use when allocating IP addresses.
```

Comment:
Each host has its own CIDR (`10.0.0.1/16`, `10.1.0.1/16`) that Romana will use when allocating IP addresses to OpenStack guests on that host.